libs/core/src/components/pds-button/pds-button.tsx

```diff
@@ -4,9 +4,9 @@ import { hasShadowDom } from '../../utils/utils';
 import { caretDown } from '@pine-ds/icons/icons';
 
 /**
- * @part button - The main button element that represents the button component.
- * @part caret - The caret icon element that appears when the button variant is 'disclosure'.
- * @part icon - The icon element that appears before the text in the button, if provided.
+ * @part button - Button element styles.
+ * @part caret - Caret icon styles. Appears only on the disclosure variant.
+ * @part icon - Icon element styles.
```

Comment:
Is `styles` the correct term here? 

mdn reference: https://developer.mozilla.org/en-US/docs/Web/API/Element/part

Reply:
My thought process here is that this is used to change the part's styles, as in style declarations, since they are intended to be targeted with CSS. If that isn't clear, I can reword all instances of part explanations.

Reply:
While yes that is also a use case, however, this is intended for the code.

Reply:
I've updated the description, let me know what you think.